main.css

```diff
@@ -1186,3 +1186,41 @@ a {
     scale: 1;
   }
 }
+
```

Comment:
Please move the css code to main.scss in src due to the use of SASS. :)

Reply:
Doing it right now ......